src/views/Viewer.vue

```diff
@@ -407,6 +428,16 @@ export default {
 				}
 			}
 		},
+
+		isFullscreenMode(mode) {
+			const modalContainer = document.getElementsByClassName('modal-container').item(0)
+			if (mode) {
+				modalContainer.classList.add('modal-container-fullscreen')
+			} else {
+				modalContainer.classList.remove('modal-container-fullscreen')
```

Comment:
Please use BEM syntax :)
`modal-container--fullscreen

Reply:
Done in https://github.com/nextcloud/viewer/pull/1588/commits/660d1fc5be91e2966194dce3d8491863ff3703fc